### PR TITLE
Add 'Approval process set' label to catalog objects cards

### PR DESCRIPTION
**Pull request by lgalis**
https://issues.redhat.com/browse/SSP-1809

Mocks: https://marvelapp.com/prototype/625jh26/screen/58833387

Adds a label to indicate whether approval processes are set for the portfolio or product. 

Screenshot from 2020-12-02 14-20-41
Screenshot from 2020-12-02 14-21-31
Screenshot from 2020-12-02 14-22-04

### CODE REVIEW

File: src/smart-components/portfolio/portfolio-item.js [hunk 1]
```diff
@@ -11,34 +11,49 @@ import {
...
+                handleCheck={() => props.onSelect(props.id)}
+                isChecked={props.isSelected}
+                id={props.id}
+              />
+            )}
+          </StyledLevel>
+        </CardHeader>
+        <ServiceOfferingCardBody {...props} />
+        <CardFooter>
+          {props.metadata &&
```
Comment (Hyperkid123) [inline src/smart-components/portfolio/portfolio-item.js:43]: You can use conditionals instead: `props?.metadata?.statistics?.approval_processes >= 0 `

**Comment by Hyperkid123** [comment]
cc @sbuenafe-rh

**Comment by sbuenafe-rh** [comment]
Looks good!

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**